ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -2650,11 +2651,11 @@ spec:
     done
   query: |
     SELECT 1 WHERE NOT EXISTS (
-    SELECT 1 FROM file WHERE 
-      path LIKE '/System/Volumes/Data/System/%%'  
-      AND type = 'directory' 
-      AND directory NOT LIKE '%Drop Box%' 
-      AND   CAST( SUBSTRING( mode ,-1) AS INTEGER) & 0x2 !=0 -- mode last char is others' permissions. bitwise with 0x2 means write permissions. (which we do not want here) 
+      SELECT 1 FROM find_cmd WHERE
+        directory = '/System/Volumes/Data/System'
+        AND type = 'd'
+        AND perm = '-2'
```

Comment:
Please keep this comment so -2 is understood.

_-- mode last char is others' permissions. bitwise with 0x2 means write permissions. (which we do not want here)_

Reply:
Sorry I deleted it again, 470170a24107eb75b7f5770df398d58c0598d904.